samples/subsys/ipc/ipc_service/remote/src/main.c

```diff
@@ -87,7 +87,7 @@ void app_task_1(void *arg1, void *arg2, void *arg3)
 		},
 	};
 
-	status = ipc_service_register_endpoint(&ept_1, &ept_cfg);
+	status = ipc_service_register_endpoint(&ept_1, &ept_cfg, NULL);
```

Comment:
To be compatible with different backends, `instance = null `should also means default backend .

Reply:
ACK